kafka/testresources/docker-compose-kraft.yaml

```diff
@@ -1,13 +1,39 @@
-version: '3'
 services:
-  kafka:
-    build: ./kraft
-    restart: always
-    ports:
-      - 9092:29092
-      - 9093:29093
+  broker-1:
+    image: apache/kafka:4.0.0
+    container_name: broker-1
     volumes:
-      - ./kraft/server.properties:/etc/kafka/server.properties
```

Comment:
Can we continue using `server.properties` it's possible by mounting them as `/mnt/shared/config/server.properties`